charts/identity-server/values.prod.yaml

```diff
@@ -437,6 +437,9 @@ services-auth-ids-api:
     CONFIGCAT_SDK_KEY: '/k8s/configcat/CONFIGCAT_SDK_KEY'
     DB_PASS: '/k8s/services-auth/api/DB_PASSWORD'
     IDENTITY_SERVER_CLIENT_SECRET: '/k8s/services-auth/IDENTITY_SERVER_CLIENT_SECRET'
+    NOVA_PASSWORD: '/k8s/gjafakort/NOVA_PASSWORD'
+    NOVA_URL: '/k8s/service-portal-api/NOVA_URL'
+    NOVA_USERNAME: '/k8s/gjafakort/NOVA_USERNAME'
```

Comment:
We probably need to create these in the production IDS account. Should create new names while we're at it, can devops to copy the values.

Reply:
Renamed and new secrets created on staging and ids-prod.